### DISCUSSION TITLE
Simulating a torsional load on a RVE

### DISCUSSION THREAD

**Discussion by Erfaaannn** [damask-multiphysics/DAMASK#5] [Q&A]
Hello everyone,
I'm trying to simulate a torsional load with the spectral solver of DAMASK 3. For this purpose, I generated my shear load with the deformation gradient rate tensor as follows:
...
- discretization:
      t: 10
      N: 50
    boundary_conditions:
      mechanical:
        dot_F: [[0, 0, 0],
                [1.e-2,      0, 0],
                [0,      0, 0]]
- discretization:
      t: 80
      N: 50
    boundary_conditions:
      mechanical:
        dot_F: [[0, 0, 0],
                [1.e-2,      0, 0],
                [0,      0, 0]]
- discretization:
      t: 10
      N: 50
    boundary_conditions:
      mechanical:
        dot_F: [[0, 1.e-2, 0],
                [0,      0, 0],
                [0,      0, 0]]

I want to investigate the initial and final pole figure during this simulation. In the initial texture the [100] crystallographic direction is parallel to the Y axis (as shown in the attached image). I expect the final pole figure of my simulation to rotate about 90 degrees just like the example image that I attached in this message. But the problem is that I don't know how to simulate such a torsional deformation with the load files of spectral solvers of DAMASK. Let me tell you that I tried a lot to increase the time of applying the deformation gradient rate. But it appears that there is a limitation in the simulations for very large deformations and usually the error of maximum cut back of DAMASK happens when I increase the time. I will be so thankful if anyone can help me with this problem.

![torsional load simulation](https://github.com/eisenforschung/DAMASK/assets/128809696/cc2e2fca-0b1c-45d0-9ba9-c8183fc2c368)

**Comment by MarDiehl** [comment]
unfortunately, torsional load with the spectral/grid solver is not possible.

**Reply by eisenlohr** [reply]
The reason for this inability lies in the fact that the spectral/grid solver always operates with periodic boundary conditions, which are incompatible with torsion of a unit cell/RVE. In other words, a unit cell that underwent torsional deformation cannot tile space anymore.
